### ISSUE TITLE
 render() error with Django 2.1

### ISSUE THREAD

**Issue by michaelqz** [jmrivas86/django-json-widget#9]
* django-json-widget version: 0.1.1
* Django version: 2.1
* Python version: 3.6.5
* Operating System: Linux

### Description

Trying to use the widget in Django 2.1. When using Django 2.0.8 - it works fine. I noticed that froala also had/has this issue-> [this may be the cause](https://github.com/froala/django-froala-editor/issues/55#issuecomment-406581533).

### What I Did

Error Occurs: `render() got an unexpected keyword argument 'renderer'`

```
Request Method: | GET
-- | --
http://xxxxxxxxxxxxxxxxxxxxxxxxxxx.vfs.cloud9.ap-southeast-1.amazonaws.com/admin/api/inftoken/test_5Fdb59148644095ac4df829c4378125001d3462a6cf4565b54b7d16e90bb/change/
2.1
TypeError
render() got an unexpected keyword argument 'renderer'
/usr/local/lib64/python3.6/site-packages/django/forms/boundfield.py in as_widget, line 93
/usr/bin/python36
3.6.5
['/home/ec2-user/environment/api',  '/usr/lib64/python3.6',  '/usr/lib64/python3.6/lib-dynload',  '/home/ec2-user/.local/lib/python3.6/site-packages',  '/usr/local/lib64/python3.6/site-packages',  '/usr/local/lib/python3.6/site-packages',  '/usr/lib64/python3.6/dist-packages',  '/usr/lib/python3.6/dist-packages']
Sat, 25 Aug 2018 17:38:07 +1000
```

Trace:

![screen](https://user-images.githubusercontent.com/19227066/44616161-10792f80-a88e-11e8-8c3c-761e65f40808.png)

**Comment by moas** [comment]
render method signature was changed in django 2.1
My solution for quickly fix create your local widget base on django-json-widget and override method

**Comment by xeor** [comment]
This is causing issues for us as well... @maos.. If you have a fix? Can you share it? Maybe a pull-request?

**Comment by xeor** [comment]
https://github.com/jmrivas86/django-json-widget/pull/10

**Comment by jmrivas86** [comment]
fixed in PR #8
